src/main/java/seedu/address/model/Mathutoring.java

```diff
@@ -0,0 +1,130 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.student.Student;
+import seedu.address.model.student.UniqueStudentList;
+
+/**
+ * Wraps all data at the mathutoring level
+ * Duplicates are not allowed (by .isSameStudent comparison)
+ */
+public class Mathutoring implements ReadOnlyMathutoring {
```

Comment:
Same case as ReadOnlyAddressBook. Take into account of people who takes over our role to maintain the code, this name will only confuse them.

Reply:
I don't think they will be confused if they read our user guide and developer guide.